code/modules/client/client_procs.dm

```diff
@@ -249,7 +249,10 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 		GLOB.preferences_datums[ckey] = prefs
 	prefs.last_ip = address				//these are gonna be used for banning
 	prefs.last_id = computer_id			//these are gonna be used for banning
-	fps = prefs.clientfps
+	if(prefs.clientfps)
+		fps = prefs.clientfps
+	else
+		fps = world.fps * 3 // 60 fps at 20 tps (tg default)
```

Comment:
Is there a reason this needs to be tied to world fps? If we for some reason changed world fps I don't see why the client fps should also change

Reply:
going to revert this part and just update player settings instead.